pages/api/input-label.md

```diff
@@ -27,7 +27,7 @@ import InputLabel from '@material-ui/core/InputLabel';
 | <span class="prop-name">FormLabelClasses</span> | <span class="prop-type">object</span> |   | `classes` property applied to the [`FormLabel`](/api/form-label/) element. |
 | <span class="prop-name">margin</span> | <span class="prop-type">enum:&nbsp;'dense'<br></span> |   | If `dense`, will adjust vertical spacing. This is normally obtained via context from FormControl. |
 | <span class="prop-name">required</span> | <span class="prop-type">bool</span> |   | if `true`, the label will indicate that the input is required. |
-| <span class="prop-name">shrink</span> | <span class="prop-type">bool</span> |   | If `true`, the label is shrunk. |
+| <span class="prop-name">shrink</span> | <span class="prop-type">enum:&nbsp;true&nbsp;&#124;<br>&nbsp;false&nbsp;&#124;<br>&nbsp;'auto'<br></span> |   | If `true`, the label is shrunk. `'auto'` can be used to restore default behaviour and leave shrink handling to FormControl. |
```

Comment:
This description could be a bit confusing, because by itself, `undefined` provides the default behavior. It needs some context around use with adornments.

Reply:
@mbrookes could you please elaborate about `undefined`? I tried passing `undefined` manually to the `InputLabel` but it seemed like nothing is changing, maybe I missed something and that could just do the trick?

Reply:
> I tried passing undefined manually to the InputLabel but it seemed like nothing is changing

What I mean is, this description doesn't explain when / why you would need to use 'auto', given that without an adornment this is already the default behavior. It doesn't mention adornments, so would be confusing read in isolation from this PR.